packages/plugins/Tips/src/components/NFTSection/index.tsx

```diff
@@ -144,6 +146,12 @@ export function NFTSection({ className, onEmpty, ...rest }: Props) {
         setNonFungibleTokenId(tokens[0].tokenId)
     }, [account, pluginID, chainId, Token?.addNonFungibleCollection])
 
+    // fetched tokens are all filtered out, keep fetching next page
+    useEffect(() => {
+        if (tokens.length) return
+        next()
+    }, [!tokens.length, fetchedTokens.length, next])
```

Comment:
```suggestion
    }, [!tokens.length, next])
```

Reply:
As the comment says, fetched tokens are all filtered out, we need to track fetchedTokens's length

Reply:
`tokens` is the filtering result of `fetchedTokens`, so if `tokens.length` is truthy, then `fetchedTokens.length` is also truthy.